### PR TITLE
[SM]Update Options.py

### PR DESCRIPTION
**Pull request by ScootyPuffJr1**
Adding documentation to Super Metroid game options.

### CODE REVIEW

**Comment by Berserker66** [comment]
@lordlou do these look good to you?

File: worlds/sm/Options.py [hunk 1]
```diff
@@ -62,39 +65,46 @@ class MaxDifficulty(Choice):
...
 
 class PowerBombQty(Range):
+    """The higher the number the higher the probability of choosing power bombs when placing a minor."""
     display_name = "Power Bomb Quantity"
     range_start = 10
     range_end = 90
     default = 10
 
 class MinorQty(Range):
+    """From 7%: minimum number of minors required to finish the game to 100%: all minors locations contain a minor (vanilla like)."""
```
Comment (lordlou) [inline worlds/sm/Options.py:100]: Maybe remove "all minors locations contain a minor (vanilla like)". That's not the case in AP. "Majors Split" is always set to full.

**Review by lordlou — COMMENTED** [review]
Other than the noted small change, that looks fine with me.
Thanks!

**Comment by ScootyPuffJr1** [comment]
I changed the wording of MinorQty and also fixed some grammar for MaxDifficulty.

**Pull request merged**